github_pr.py

```diff
@@ -75,6 +76,81 @@ def _load_issue(**args):
     return repo.get_issue(args['number'])
 
 
+def __validate_list_of_dict(list_of_dict):
+    """
+    Validates that obj is a list containing dictionaries with entries for 'pr' and 'issue'
+    """
+    if list_of_dict:
+        if isinstance(list_of_dict, list) and 'pr' in list_of_dict[0] and 'issue' in list_of_dict[0]:
+            return True
+        else:
+            return False
+    else:
+        return False
```

Comment:
cool didnt know the return would do logic? im assuming thats what this shortened use is, right?